gitbutler-ui/src/lib/components/FileContextMenu.svelte

```diff
@@ -29,13 +34,58 @@
 					<ContextMenuItem label="Discard changes (Binary files not yet supported)" disabled />
 				{:else}
 					<ContextMenuItem
-						label="Discard changes"
+						label={item.files.length === 1 ? 'Discard file' : 'Discard files'}
 						on:click={() => {
 							confirmationModal.show(item);
 							dismiss();
 						}}
 					/>
 				{/if}
+				{#if item.files.length === 1}
+					<ContextMenuItem
+						label="Copy Path"
+						on:click={async () => {
+							try {
+								if (!project) return;
+								const absPath = await join(project.path, item.files[0].path);
+								navigator.clipboard.writeText(absPath);
+								dismiss();
+							} catch (err) {
+								console.log('Failed to copy path', err);
+								toasts.error('Failed to copy path');
+							}
+						}}
+					/>
+					<ContextMenuItem
+						label="Copy Relative Path"
+						on:click={() => {
+							try {
+								if (!project) return;
+								navigator.clipboard.writeText(item.files[0].path);
+								dismiss();
+							} catch (err) {
+								console.log('Failed to copy relative path', err);
+								toasts.error('Failed to copy relative path');
+							}
+						}}
+					/>
+				{/if}
+				<ContextMenuItem
+					label="Open in VSCode"
+					on:click={async () => {
+						try {
+							if (!project) return;
+							for (let file of item.files) {
+								const absPath = await join(project.path, file.path);
+								open(`vscode://file${absPath}`);
+							}
+							dismiss();
+						} catch {
+							console.log('Failed to open in VSCode');
```

Comment:
Can we change this to  catch the error, switch log to `console.error()` and pass the err as a second argument?